src/ape/utils/abi.py

```diff
@@ -285,45 +285,43 @@ def decode_value(self, abi_type, value):
 
         return value
 
+def parse_type(type: Dict[str, Any]) -> Union[str, Tuple, List]:
```

Comment:
It looks like, based on some failing tests, that this is not handling everything. For example, Solidity structs with nested arrays.

See some of the tests in `tests/functional/test_contract_instance()`!

Reply:
Managed to make the testing suite work on my Mac (I had problems on Linux Ubuntu).

But now when running the testing suite, it is using `ape` that sits in site-packages (in dependencies). So basically, it's not testing the `ape` in the repo but the one in deps. Why does `ape` install itself as a dep?

How are you running the tests? I go `python -m pytest` in root dir (after activating the `venv` and installing the deps with `pip instsall -e .'[dev]'`)

Reply:
the very first run of `python -m pytest` after installing a fresh `venv` does use the repo `ape`. However, subsequent uses start using `ape` from deps.

Reply:
Are you installing in editable mode?

`pip install -e path/to/ape`

Reply:
`pip install -e .'[dev]'` like it says in `CONTRIBUTING.md`

Reply:
I just commented out github test as well as GitHubClient import somewhere, to just silence the error. That made it work for my Ubuntu machine, which is consistently using the correct ape for tests.